api/src/main/java/org/openmrs/Location.java

```diff
@@ -774,4 +776,20 @@ public String getAddress15() {
 	public void setAddress15(String address15) {
 		this.address15 = address15;
 	}
+	
+	/**
```

Comment:
Can you include the full method descriptions?

Reply:
working on it?

Reply:
hello @dkayiwa! i have improved the description and changed the version to the current release